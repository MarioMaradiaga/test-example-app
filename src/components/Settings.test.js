import Settings from "./Settings";
import renderer from "react-test-renderer";

test("should render Settings page", () => {
  const tree = renderer.create(<Settings />).toJSON();
  expect(tree).toMatchSnapshot();
});
