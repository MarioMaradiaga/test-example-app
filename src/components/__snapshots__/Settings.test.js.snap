// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Settings page 1`] = `
<div
  className="settings"
>
  <h1>
    Settings
  </h1>
</div>
`;
